core/src/main/java/we/retail/core/components/impl/CartEntryServlet.java

```diff
@@ -76,16 +77,22 @@
     private static final String CONTENT_WE_RETAIL_DEFAULT = "/content/we-retail/us/en/";
     private static final String CART_PATH = "/user/cart/jcr:content/root/responsivegrid/cart";
     private static final String CART_PRICES_PATH = "/user/cart/jcr:content/root/responsivegrid/shoppingcartprices";
+
+    @SuppressWarnings("CQRules:CQBP-71")
```

Comment:
Is there a documentation available for this rule?

Reply:
There's no externally available documentation for this rule as of yet.

Reply:
The documentation for this rule is now live: https://helpx.adobe.com/experience-manager/cloud-manager/using/custom-code-quality-rules.html#AvoidHardcodedappsandlibsPaths